assignment4.Rmd

```diff
@@ -81,22 +87,25 @@ plot(pca, type = "lines")
 ```{r}
 #Now, create a data frame of the transformed data from your pca.
 
-D3 <- as.data.frame(pca$x)
+D3 <- as.data.frame(pca$x[,1:5])
 
 #Attach the variable "mean_correct" from your original data frame to D3.
 
 D4 <- cbind(D3, as.data.frame(D1$mean_correct))
-
+COR2<-cor(D4)
 #Now re-run your scatterplots and correlations between the transformed data and mean_correct. If you had dropped some components would you have lost important infomation about mean_correct?
-
-COR2 <- 
+corrplot(COR2, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
+# no, because component 6 can only explain a little variance.
```

Comment:
Try to run the `corrplot()` with PC6 in as well. You will find that although PC6 accounts for the least variance, it happens to be variance that correlates the strongest with your outcome. There is a danger in using this method, that you may throw away important information even though it is the least amount of information.